resource_cloud/views.py

```diff
@@ -450,10 +452,12 @@ def patch(self, provision_id):
                     self.delete(provision_id)
             else:
                 pr.state = args['state']
+                if pr.state == 'running' and user.is_admin:
```

Comment:
Right, thought first this might be semi-intentional as admin user is required to do so, but it's better this way.